### ISSUE TITLE
resource.dialect became required between 4.11.0 and 4.14.0?

### ISSUE THREAD

**Issue by karlcz** [frictionlessdata/frictionless-py#888]
# Overview

We have existing datapackages which validated (zero errors) with `frictionless validate datapackage.json` using frictionless-py 4.11.0 but fail when we upgraded to 4.14.0 with errors like this:

```
package-error  The data package has an error: "'doubleQuote' is a required property" at \
"resources/14/dialect" in metadata and at "properties/resources/items/properties/dialect/required" \
in profile
```
Is this a regression or an expected change in requirements?  Reading the spec online, I do not see any mention of `dialect` even as an optional field for resources at https://specs.frictionlessdata.io/data-resource/ linked from https://specs.frictionlessdata.io/data-package/

The resources in our case have a `"path": "foo.tsv"` local TSV file and  we typically have not supplied `format` or `mediatype` properties either.

Edit: I should also add that the error message is misleading as it complains about `doubleQuote` when the enclosing document and `dialect` property are absent.

---

Please preserve this line to notify @roll (lead of this repository)

**Comment by roll** [comment]
Thanks @karlcz,

It looks like a bug. I'll investigate. Can you please share a data package example?

**Comment by karlcz** [comment]
Sure, this is slightly complicated as an example, but is synthetic data we have on hand that exhibits the problem...
[frictionless-dialect-error.zip](https://github.com/frictionlessdata/frictionless-py/files/6903953/frictionless-dialect-error.zip)
there are two JSON files in there, one without dialect definitions which fails, and one which adds diaect definitions and then validates cleanly for me.

**Comment by roll** [comment]
@karlcz 
It was a regression in JSONSchema profile for `tabular-data-package`. I'm releasing a fix with `frictionless@4.14.1`. Please try it when it's on PyPi.

In general, I would not recommend using `tabular-data-package` profile. It doesn't add any additional value over `data-package` if you use `tabular-data-resource` for resources.
